starport/interface/cli/starport/cmd/network_chain_list.go

```diff
@@ -43,6 +87,56 @@ func networkChainListHandler(cmd *cobra.Command, args []string) error {
 		chainTable.Append([]string{chainSummary.ChainID, chainSummary.Source, validators, proposals})
 	}
 	chainTable.Render()
+}
+
+// listChainSummaries lists chains with their summary info by using nextPageKey as the
+// pagination key to fetch the next page.
+func listChainSummaries(nb *networkbuilder.Builder, ctx context.Context, pageKey []byte) (summaries []ChainSummary,
+	nextPageKey []byte, err error) {
+	var chains []spn.Chain
+	chains, nextPageKey, err = nb.ChainList(ctx, spn.PaginateChainListing(pageKey, 40))
```

Comment:
Little detail: putting this as a constant for better maintainability

Reply:
solved by f4baf09.